api/Attr.json

```diff
@@ -41,6 +41,9 @@
           "webview_android": {
             "version_added": true,
```

Comment:
This whole file needs doing by hand, 

Supports needs to be in alphabetical order,

Check that the information for Chrome is valid in Samsung Internet and add entry manually.

Maybe also check Chrome Android as well to update that data.

Reply:
Alphabetic sorting fixed. Rest still needs checking.